tests/drivers/tee/tee/boards/native_posix_64.overlay

```diff
@@ -0,0 +1,9 @@
+/ {
+	firmware {
+		 tee {
+		       compatible = "test,tee";
+		       method = "smc";
```

Comment:
It would be cool to have both hvc and smc in tests of course for different boards
one test for bare-metal gen3/gen4 boards and another for xen_dom0 shield

Reply:
BTW it can be a good idea to mention about zephyr-optee-tests app somewhere in documentation. What do you think?

Reply:
yep, I think it is a good idea